lighthouse-core/test/lib/drivers/driver-test.js

```diff
@@ -80,4 +83,123 @@ describe('Browser Driver', () => {
     assert.notEqual(opts.url, req1.url, 'opts.url changed after the redirects');
     assert.equal(opts.url, req3.url, 'opts.url matches the last redirect');
   });
+
+  it('will fail when multiple tabs are found with the same active serviceworker', () => {
```

Comment:
I think it would be helpful to surround these tests with a nested
`describe('Multiple tab check', () => {` to make it clear what the `it` in these are, since these are just tests for `checkForMultipleTabsAttached`, not end to end tests.